ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/internal/results/TransactionInfoResult.java

```diff
@@ -39,7 +37,7 @@ public String getHash() {
 
   @JsonGetter(value = "addedToPoolAt")
   public String getAddedToPoolAt() {
-    return addedToPoolAt.toString();
+    return Long.toString(addedToPoolAt);
```

Comment:
This is wrong - Instant.toString outputs a ISO8601 formatted date time but this outputs a number.